### ISSUE TITLE
Directus badges

### ISSUE THREAD

**Issue by MichaelHinrichs** [badges/shields#10342]
### 📋 Description

For Directus.
![Static Badge](https://img.shields.io/badge/item%20id-1-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/item%20status-published-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/item%20title-Hello%2C%20world!-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/item%20body-This%20is%20my%20first%20article-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/item%20featured%20image-768eabec--3c54--4110--a6bb--64b548116661-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/item%20author-0bc7b36a--9ba9--4ce0--83f0--0a526f354e07-263238?logo=directus)

![Static Badge](https://img.shields.io/badge/file%20id-4f4b14fa--a43a--46d0--b7ad--90af5919bebb-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20storage-local-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20filename%20disk-4f4b14fa--a43a--46d0--b7ad--90af5919bebb.jpeg-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20filename%20download-paulo--silva--vSRgXtQuns8--unsplash.jpg-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20title-Paulo%20Silva%20(via%20Unsplash)-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20type-image/jpeg-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20folder-null-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20uploaded%20by-0bc7b36a--9ba9--4ce0--83f0--0a526f354e07-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20uploaded%20on-2021--02--04T11:37:41--05:00-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20modified%20by-null-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20modified%20on-2021--02--04T11:37:42--05:00-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20filesize-3442252-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20width-3456-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20height-5184-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20focal%20point%20x-null-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20focal%20point%20y-null-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20duration-null-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20description-null-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20location-null-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20tag-photo-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20tag-pretty-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20icc%20version-2.1-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20icc%20intent-Perceptual-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20icc%20cmm-lcms-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20icc%20deviceClass-Monitor-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20icc%20colorSpace-RGB-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20icc%20connectionSpace-XYZ-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20icc%20platform-Apple-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20icc%20creator-lcms-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20icc%20description-c2-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/file%20icc%20copyright--263238?logo=directus)

![Static Badge](https://img.shields.io/badge/activity%20action-create-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/activity%20collection-articles-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/activity%20comment-null-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/activity%20id-5-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/activity%20ip-139.178.128.0-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/activity%20item-1-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/activity%20timestamp-2021--02--02T12:50:26--05:00-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/activity%20user-2d321940--69f5--445f--be6b--c773fa58a820-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/activity%20user%20agent-Mozilla/5.0%20(Macintosh;%20Intel%20Mac%20OS%20X%2010_15_6)%20AppleWebKit/605.1.15%20(KHTML,%20like%20Gecko)%20Version/14.0.2%20Safari/605.1.15-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/activity%20revisions-[4]-263238?logo=directus)

![Static Badge](https://img.shields.io/badge/Content%20Version%20id-21a7ed5f--eb19--42ae--8ee2--61f25b8c4eb5-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/Content%20Version%20key-my_version-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/Content%20Version%20name-My%20Version-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/Content%20Version%20collection-my_collection-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/Content%20Version%20item-1-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/Content%20Version%20date%20created-2023--08--23T10:38:20.686Z-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/Content%20Version%20user%20created-a2dbc923--7c75--4d26--83f4--4674bfa7be81-263238?logo=directus)

![Static Badge](https://img.shields.io/badge/dashboard%20name-My%20Dashboard-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/dashboard%20icon-space%20dashboard-263238?logo=directus)

![Static Badge](https://img.shields.io/badge/flow%20id-2fab3b9d--0543--4b87--8a30--3c5ee66fedf1-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/flow%20name-My%20Flow-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/flow%20icon-bolt-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/flow%20color-%23112233-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/flow%20description-Description%20for%20my%20flow-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/flow%20status-active-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/flow%20trigger-manual-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/flow%20accountability-$trigger-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/flow%20date%20created-2022--05--11T13:14:52Z-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/flow%20user%20created-12e62fd0--29c7--4fd3--b3d3--c7a39933e8af-263238?logo=directus)
![Static Badge](https://img.shields.io/badge/flow%20operation-92e82998--e421--412f--a513--13701e83e4ce-263238?logo=directus)

### 🔗 Data

https://docs.directus.io/reference/introduction.html
Yes there is an API Key.

### 🎤 Motivation

Directus data at a glance. Directus even supports [Markdown](https://docs.directus.io/app/data-model/fields/text-numbers.html#markdown).

**Comment by chris48s** [comment]
Having done a bit of reading, it seems that directus isn't a centralised service. It is something where everyone has their own instance (be that self-hosted or directus cloud).
Also API access seems to require a token attached to a user that grants access to that user's resources. I don't think there is any anonymous access.
I think this means this would be unsuitable for shields.io as every instance/user would have to supply a token in a GET param and it looks like this would be difficult to do safely.

**Comment by PyvesB** [comment]
Agreed with what Chris said, will go ahead and close the issue. Thanks for the suggestion, though!
